README.md

```diff
@@ -1,26 +1,190 @@
+```
+#### Upgrade to latest version
+```
+pip3 install -U sfctl
+```
+#### Upgrade to specific version
+```
+pip3 install -U sfctl==replace-with-required-version
+```
+#### Uninstall
+```
+pip uninstall -y sfctl
+```
+## Installing multiple versions side-by-side using python virtual environment
+There may be a need to install different versions of sfctl side by side to interact with clusters running different versions of Service Fabric runtime. 
+
+[pipenv](https://docs.pipenv.org/) package can be used to setup a python virtual environment that can be used to install a different version of sfctl separate from the globally installed version.
 
-### Launching
+1. Install pipenv.
 
-```bash
-sfctl <command>
+    **NOTE:** Only this command requires an admin command prompt or sudo access. Do not use admin command prompt or sudo for rest of the commands below
+    ```
+    pip3 install pipenv
+    ```
+2. Create a directory for installing files for virtual environment (commands below installs sfctl 1.1.0)
+    ```
+    mkdir sfctl-1.1.0
+    ```
+3. Install sfctl in virtual env
+    ```
+    pipenv --three install sfctl-1.1.0
+    ```
+4. Create a shell that uses files in virtual environment
+    ```
+    pipenv --three shell
+    ```
+5. Run 1.1.0 of sfctl
+    ```
+    pip show sfctl
+    sfctl --help
+    ```
+6. Exit virtual environment shell
+    ```
+    exit
+    ```
+7. Run globally installed version of sfctl
+    ```
+    pip show sfctl
+    sfctl --help
+    ```
+
+## Installing multiple versions side-by-side in a Docker container
+sfctl github repo has Dockerfile to generate a container with the required version of sfctl.
+
+1. Install [Docker CE](https://www.docker.com/community-edition) the machine. 
+
+2. Clone [sfctl repo](https://github.com/Azure/service-fabric-cli) using information on the github page - https://github.com/Azure/service-fabric-cli
+
+2. Open a Command Prompt (Windows) or Terminal window (Linux or Mac) and navigate to the Dockerfiles folder under the root of the enlistment
+
+    ```
+    pushd path-to-root-of-enlistment/Dockerfiles
+    ```
+
+3. Build the docker image
+
+    Windows
+    ```
+    REM replace 2.0.0 with required version
+    docker build -t sfctl_2.0.0 -f Dockerfile.windows --build-arg sfctl_version=2.0.0 .
+    ```
+    Linux
+    ```
+    # replace 2.0.0 with required version
+    docker build -t sfctl_2.0.0 -f Dockerfile.linux --build-arg sfctl_version=2.0.0 .
+    ```
+
+4. Start the container in interactive mode. Replace the name of the name of the image with the image for the required version
+    ```
+    docker run -it sfctl_2.0.0
+    ```
+5. Once the container starts at the container command prompt run sfctl
+    ```
+    sfctl -h
+    ...
+    ```
+6. Exit the container
+    ```
+    exit
+    ```
+## Command line options and commands
+ sfctl has extensive help for command line options and commands. --help option gives detailed information
+
+Output from sfctl 2.0.0
 ```
+sfctl --help

```

Comment:
This will become stale quickly, it's okay to just specify the `-h` or `--help` flag.